### ISSUE TITLE
mantl-dns install has a dependency issue with NetworkManager while running playbook

### ISSUE THREAD

**Issue by KaGeN101** [mantl/mantl#1929]
The dependency on NetworkMananger fails on all nodes when trying to install mantl-dns

```
failed: [mantl-control-03] => {"changed": true, "rc": 1, "results": ["Loaded plugins: fastestmirror\nLoading mirror speeds from cached hostfile\n * base: linux.cc.lehigh.edu\n * epel: s3-mirror-us-east-1.fedoraproject.org\n * extras: mirrors.lga7.us.voxel.net\n * updates: distro.ibiblio.org\nResolving Dependencies\n--> Running transaction check\n---> Package mantl-dns.x86_64 0:1.1.0-5.centos will be installed\n--> Processing Dependency: NetworkManager = 1:1.0.6 for package: mantl-dns-1.1.0-5.centos.x86_64\n--> Finished Dependency Resolution\n You could try using --skip-broken to work around the problem\n You could try running: rpm -Va --nofiles --nodigest\n"]}
msg: Error: Package: mantl-dns-1.1.0-5.centos.x86_64 (asteris-mantl-rpm)
           Requires: NetworkManager = 1:1.0.6
           Available: 1:NetworkManager-1.4.0-12.el7.x86_64 (base)
               NetworkManager = 1:1.4.0-12.el7
           Available: 1:NetworkManager-1.4.0-13.el7_3.x86_64 (updates)
               NetworkManager = 1:1.4.0-13.el7_3
           Available: 1:NetworkManager-1.4.0-14.el7_3.x86_64 (updates)
               NetworkManager = 1:1.4.0-14.el7_3
           Available: 1:NetworkManager-1.4.0-17.el7_3.x86_64 (updates)
               NetworkManager = 1:1.4.0-17.el7_3
           Available: 1:NetworkManager-1.4.0-19.el7_3.x86_64 (updates)
               NetworkManager = 1:1.4.0-19.el7_3
```

- No changes to sanple.yaml
- Ansible version (`ansible --version`): ansible 1.9.6
- Python version (`python --version`):Python 2.7.12
- Git commit hash or branch: f19d567faee19c1399ad4fafac4830b235d241b1
- Cloud Environment: AWS
- Terraform version (`terraform version`): Terraform v0.9.3

**Comment by Theaxiom** [comment]
I just updated the Vagrant box and am seeing this issue as well.

**Comment by Theaxiom** [comment]
Closing this. This is a duplicate of #1859
